lang/cs/Org.Apache.REEF.IO/FileSystem/Local/LocalFileSystem.cs

```diff
@@ -28,18 +28,19 @@ namespace Org.Apache.REEF.IO.FileSystem.Local
     /// </summary>
     internal sealed class LocalFileSystem : IFileSystem
     {
-        private readonly string _uriPrefix;
-
         [Inject]
         private LocalFileSystem()
         {
-            _uriPrefix = "";
         }
 
-        public string UriPrefix
+        /// <summary>
+        /// Create Uri from given file name
+        /// </summary>
+        /// <param name="path"></param>
+        /// <returns></returns>
+        public Uri CreateUriForPath(string path)
```

Comment:
Drop the documentation here and document the interface instead. Then, add what's different here.